### ISSUE TITLE
Missing emcc --help documentation on emcc flags that affect wasm codegen?

### ISSUE THREAD

**Issue by juj** [emscripten-core/emscripten#12760]
I was looking for the flag to enable bulk memory operations in emcc, but took a bit of work to find it.

There is also the `-mnontrapping-fptoint` flag that one can pass to `emcc`, that does not seem to be documented in `emcc --help`.

Finally stumbled to https://clang.llvm.org/docs/ClangCommandLineReference.html#webassembly that lists all(?) the flags, but does not explain them.

Would it make sense to collect all these specific flags under `emcc --help`, or some other page, along with an explainer, which browser and shell versions adopted support for them and when, and e.g. a link to the spec?

**Comment by kripken** [comment]
Good question, I'm not sure of the status of docs on the wasm backend. cc @tlively @aheejin

**Comment by sbc100** [comment]
I think if we make improvements to the docs we should probably do it upstream in llvm and them somehow have `emcc --help` extract that info from clang's `--help` (or clang's docs).  i.e. lets avoid documenting what should be documented upstream.

**Comment by aheejin** [comment]
I agree w/ @sbc100 that the doc improvements should be made upstream. But there is not much to say even there other than "Enable *** feature" / "Disable *** feature" though, unless we want to explain that those features are...

Also not sure if we need to extract that into emcc docs. emcc mostly has its own feature flags, which should in turn turn on or off clang features..?
